Cargo.toml

```diff
@@ -66,6 +66,9 @@ pprof = { version = "0.9", features = ["criterion", "flamegraph"] } # MSRV 1.56
 bench = false
 
 [features]
+default = ["halo2-batch", "halo2-multicore"]
+halo2-batch = ["halo2_proofs/batch"]
```

Comment:
Per above, I think this can be removed:
```suggestion
```

However, if we _do_ retain it (and allow `halo2_proofs/batch` to be disabled), then `orchard::circuit::Proof::add_to_batch` needs to be placed behind this feature flag. And I agree with @daira that in this case, the feature flag should be the same as the upstream feature (so `batch`).